.eslintrc.yml

```diff
@@ -2,6 +2,12 @@ extends: airbnb-base
 plugins:
   - import
 rules:
+  no-underscore-dangle:
+  - error
+  - allowAfterThis: true
+  space-before-function-paren:
+  - error
+  - named: always
```

Comment:
I'm using standard rules + semicolon and gave priority to the `extends` you already had there.